endpoints/openrtb2/sample-requests/valid-native/eventtracker-exchange-specific.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This test specifically verifies end to end functionally for https://github.com/prebid/prebid-server/issues/1732